tapir/shifts/tests/test_shiftwatch_forms.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from django.urls import reverse
+
+from tapir.shifts.models import StaffingEventsChoices, ShiftWatch
+from tapir.shifts.tests.factories import ShiftFactory
+from tapir.utils.tests_utils import TapirFactoryTestBase
+
+
+class TestShiftCancel(TapirFactoryTestBase):
+
+    @pytest.mark.django_db
+    def test_shift_watch_form_creates_shift_watch_instance(self):
```

Comment:
Try to follow the [naming convention](https://github.com/SuperCoopBerlin/tapir/wiki/Automated-tests-%E2%80%90-overview#naming-convention)